src/domain/profile/Profile.tsx

```diff
@@ -19,13 +19,20 @@ import settingsIcon from '../../assets/icons/svg/gear.svg';
 import Button from '../../common/components/button/Button';
 import EditProfileModal from './modal/EditProfileModal';
 import { clearEvent, saveChildrenEvents } from '../event/state/EventActions';
+import { defaultProfileData } from './state/ProfileReducers';
 
 const Profile: FunctionComponent = () => {
   const [isOpen, setIsOpen] = React.useState(false);
   const { loading, error, data } = useQuery<ProfileQueryType>(profileQuery);
   const { t } = useTranslation();
   const dispatch = useDispatch();
 
+  useEffect(() => {
```

Comment:
I am not sure if this is the right approach to solve this: We get rid of the warning but could it be that we are just brushing the problem under the carpet?